massgen/orchestrator_collaborators/final_presentation_runner.py

```
@@ -0,0 +1,1237 @@
+                ):
+                    selected_agent.backend.end_round_tracking("post_evaluation")
+
+        # Show workspace modal in no-git mode (runs before clear_workspace below)
+        await orch._show_workspace_modal_if_needed()
+
+        # Review isolated changes if write_mode isolation was enabled
+        if orch._isolation_manager:
+            selected_agent = orch.agents.get(orch._selected_agent)
+            if selected_agent:
+                ppm = selected_agent.backend.filesystem_manager.path_permission_manager
+                for orig_path, removed_mp in orch._isolation_removed_paths.items():
```

Ruff (0.15.14)

[warning] Loop control variable `orig_path` not used within loop body

Rename unused `orig_path` to `_orig_path`

(B007)